test/dm_control_test.py

```diff
@@ -0,0 +1,101 @@
+"""Tests the functionality of the dm_control_wrapper."""
+
+import inspect
+
+import numpy as np
+import pytest
+from dm_control import suite
+from dm_control.suite import (
+    acrobot,
+    ball_in_cup,
+    cartpole,
+    cheetah,
+    dog,
+    finger,
+    fish,
+    hopper,
+    humanoid,
+    humanoid_CMU,
+    lqr,
+    manipulator,
+    pendulum,
+    point_mass,
+    quadruped,
+    reacher,
+    stacker,
+    swimmer,
+    walker,
+)
+from PIL import Image
+
+from shimmy import dm_control_wrapper
+
+# Find all domains imported.
+_DOMAINS = {
```

Comment:
I believe that Suite has a parameter that contains all of configurations. Im not a fan of using `locals`

Reply:
I haven't actually used suite before, but based on what I'm seeing on how they do it, they [use locals as well](https://github.com/deepmind/dm_control/blob/main/dm_control/suite/__init__.py#L45). I've combed through their codebase and (lack thereof) of documentation and can't seem to find that parameter you're referring to, do you have a search term/name I can use to pull at the thread?

Reply:
Edit: figured it out